nifi-nar-bundles/nifi-standard-services/nifi-dbcp-service-bundle/nifi-dbcp-service/pom.xml

```diff
@@ -81,5 +81,17 @@
             <artifactId>hamcrest-all</artifactId>
             <version>1.3</version>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
```

Comment:
Do we need to bring in Spring JDBC for testing? Seems like we should be able to test this functionality without the added dependency?